web/app/js/components/TopRoutesModule.jsx

```diff
@@ -29,26 +28,14 @@ class TopRoutesBase extends React.Component {
     return <ErrorBanner message={error} />;
   }
 
-  configureProfileMsg() {
-    return (
-      <Card>
-        <CardContent>
-          <Typography>
-            No traffic found.  Does the service have a service profile?  You can create one with the `linkerd profile` command.
-          </Typography>
-        </CardContent>
-      </Card>
-    );
-  }
-
   render() {
     const {data} = this.props;
     let metrics = processTopRoutesResults(_.get(data, '[0].routes.rows', []));
 
     return (
       <React.Fragment>
         {this.banner()}
-        {_.isEmpty(metrics) ? this.configureProfileMsg() : null}
+        {_.isEmpty(metrics) ? <ConfigureProfilesMsg /> : null}
```

Comment:
Can we update this module to display a spinner if `props.loading` is true, instead of displaying the `ConfigureProfilesMsg`? Or if it's difficult to add a spinner, maybe we can just suppress the `ConfigureProfilesMsg` component while `props.loading` is true.